crates/db_schema/src/impls/custom_emoji_keyword.rs

```diff
@@ -0,0 +1,23 @@
+use crate::{
+  schema::custom_emoji_keyword::{custom_emoji_id, dsl::custom_emoji_keyword},
+  source::custom_emoji_keyword::{CustomEmojiKeyword, CustomEmojiKeywordInsertForm},
+  utils::{get_conn, DbPool},
+};
+use diesel::{dsl::insert_into, result::Error, ExpressionMethods, QueryDsl};
+use diesel_async::RunQueryDsl;
+
+impl CustomEmojiKeyword {
+  pub async fn create(pool: &DbPool, form: &CustomEmojiKeywordInsertForm) -> Result<Self, Error> {
+    let conn = &mut get_conn(pool).await?;
+    insert_into(custom_emoji_keyword)
+      .values(form)
+      .get_result::<Self>(conn)
+      .await
+  }
+  pub async fn delete(pool: &DbPool, emoji_id: i32) -> Result<usize, Error> {
+    let conn = &mut get_conn(pool).await?;
+    diesel::delete(custom_emoji_keyword.filter(custom_emoji_id.eq(emoji_id)))
+      .execute(conn)
+      .await
+  }
+}
```

Comment:
I would get rid of this file and also get rid of CustomEmojiKeywordInsertForm, moving it all into custom_emoji.rs methods/structs. Even though they are backed by different db tables, they will only ever be written and deleted together, so it keeps the api simple.

Reply:
Okay i merged the file, but unsure what you mean by getting rid of the insertform for keyword. Where should the keyword be saved in that case? I could kinda merge it with CustomEmoji::create i guess?